stubs/python-jose/jose/backends/cryptography_backend.pyi

```diff
@@ -1,11 +1,11 @@
 from typing import Any
 
-from .base import Key as Key
+from .base import Key
 
-def decode_dss_signature(signature: bytes) -> tuple[int, int]: ...
-def encode_dss_signature(r: int, s: int) -> bytes: ...
```

Comment:
Have these deliberately been deleted?

Reply:
Yes, they don't seem to be exposed https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/backends/cryptography_backend.py